fcrepo/fcrepo
# Embedded Resource representation

**JIRA Ticket**: https://jira.lyrasis.org/browse/FCREPO-3525

# What does this Pull Request do?
It adds WebAC checks for the contained resources when asking for the `http://www.w3.org/ns/oa#PreferContainedDescriptions` 
Adds a new `ResourceTripleService` that wraps the other services, so it can be called for the contained resources too.
Cleans up the UriAwareResourceModelFactory to remove references to the deprecated IdentifierConverter.

# How should this be tested?

1. Create a container and a child container.
1. Get the container with the `return=representation; include="http://www.w3.org/ns/oa#PreferContainedDescriptions"` Prefer header. 
1. See the child resource in the output.
1. Add an ACL to the container allowing access and add an ACL to the child container disallowing access to a user.
1. Retry the above request as the user. Get denied access.

# Interested parties
Tag (@ mention) interested parties or, if unsure, @fcrepo/committers
